EpamWebSiteTestingFramework/Epam.TestAutomation.Core/Elements/BaseElement.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.ObjectModel;
+using Epam.TestAutomation.Core.DriverCreator;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace Epam.TestAutomation.Core.Elements;
+
+public abstract class BaseElement
+{
+    public IWebElement Element { get; }
+
+    protected BaseElement(By locator)
+    {
+        Element = Browser.Driver.FindElement(locator);
+    }
+
+    protected BaseElement(IWebElement element)
+    {
+        Element = element;
+    }
+
+    public string GetText() => Element.Text.Trim();
+    public string GetTextFromAttribute() => Element.GetAttribute("innerText");
```

Comment:
make GetTextFromAttribute() more abstract. (add a parameter to be able to use this method with different attributes)